corehq/toggles.py

```diff
@@ -499,6 +499,13 @@ def _ensure_valid_randomness(randomness):
     [NAMESPACE_DOMAIN]
 )
 
+DHIS2_REPEATER_INTEGRATION = StaticToggle(
```

Comment:
I'm OK with having another toggle, but I'm wondering why you chose to add a new one instead of using the existing one.

Reply:
@kaapstorm I added new toggle because when we worked on the other integrations or new features we always had a requirement to add a new one.

If you want I can remove my toggle.

Reply:
:+1: That makes sense. Don't worry about changing it. We may end up merging all the integration-related toggles into one.